### PR TITLE
Update environment.launch

### PR DESCRIPTION
**Pull request by itabrl**
# PR Details
## Description

set gpu flag to true
## Related Issue
894

## How Has This Been Tested?
deployed on White Pacifica 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [x] I have added any new packages to the sonar-scanner.properties file
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by aswath1 — APPROVED** [review]

**Review by msmcconnell — CHANGES_REQUESTED** [review]
Just blocking Merge until we can hammer out the vm vs vehicle discussion

**Review by msmcconnell — CHANGES_REQUESTED** [review]
For this to work properly you need to be passing in this argument from the config, not hard coding it. Otherwise the VM will still try to use cuda

**Review by msmcconnell — CHANGES_REQUESTED** [review]

**Review by msmcconnell — APPROVED** [review]
Will merge when all 3 prs are complete

**Pull request merged**
